packages/types/src/types.ts

```diff
@@ -638,6 +641,9 @@ export const Optional = <
     v => v
   )
 
+/**
+ * @deprecated Reference values cannot be imported into the Cadence interpreter, will be removed in future versions
+ */
 export const Reference = typedef(
```

Comment:
Don't know what the deal is with this... if it's a relic of the past or just a mistake altogether, because `Reference` values cannot be imported into the Cadence interpreter, so this will never work.